src/pages/bioclimatic/index.js

```diff
@@ -23,11 +23,11 @@ const Container = () => {
   const { data } = useScenariosPerCountry(country);
 
   // parsing
-  const scenarios = data && data.scenarios;
+  const scenarios = data && data.scenarios && data.scenarios.filter(s => s.key !== 'current');
   const parsedScenarios =
     scenarios &&
     scenarios.map(sc => ({
-      label: sc.name,
+      label: `${sc.name} - ${sc.key.replace('rcp45', 'RCP 4.5').replace('rcp85', 'RCP 8.5')}`,
```

Comment:
To be honest I don't think it's a good approach here. Why we just don't change the data in GraphCMS instead of doing hardcoded parses with regexp, etc? Maybe change the `name` values in the `Scenario` table from `As usual` to `RCP 4.5` and from `Optimistic` to `RCP 8.5`. It will work for timeline, for bioclimatic filters you will need to add only `Business as usual` and  `Slowly Declining Emissions` on the front end side. Or the second approach, maybe you can add `short-name` field to the `Scenario` table, where you put `RCP 4.5` and `RCP 8.5`, and in the `name` field put `Business as usual` and  `Slowly Declining Emissions` fields. If we're able to change this bit in GraphCMS, we should definitely do it.

Reply:
On it 👍